eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/PointerStatements.scala

```diff
@@ -35,6 +35,14 @@ object PointerStatements {
 
   object Insert {
 
+    def apply[F[_]: Parallel](
+      legacy: Insert[F],
```

Comment:
could we abstract away from `legacy` & `other`, I'd better use `first` / `second` ?